.metamaskrc.dist

```diff
@@ -6,6 +6,8 @@ INFURA_PROJECT_ID=00000000000
 
 ;PASSWORD=METAMASK PASSWORD
 ;SEGMENT_WRITE_KEY=
+;SEGMENT_DELETE_API_SOURCE_ID=
```

Comment:
Could you clarify what these environment variables are?

It was my understanding that we're using our own API for this, not using Segment directly. Our API will also be requesting deletion from more than just Segment. If that's the case, making these environment variables named `SEGMENT_` seems inaccurate

Reply:
updated